vital/types/metadata.h

```diff
@@ -209,23 +209,26 @@ class VITAL_EXPORT metadata_item
     : public metadata_item
   {
   public:
-    // -- CONSTRUCTORS --
     unknown_metadata_item()
-      : metadata_item( "Requested metadata item is not in collection", 0, VITAL_META_UNKNOWN )
+      : metadata_item( "<UNKNOWN>", unknown_t{}, VITAL_META_UNKNOWN )
```

Comment:
I'm fine with changing the name but curious about your rationale?

Reply:
Looking back on it now that I understand things better, I don't think there is sufficient reason to change it. I'll restore the original version.